src/inspect_ai/util/_sandbox/__init__.py

```diff
@@ -8,11 +8,14 @@
     SandboxEnvironmentSpec,
     SandboxEnvironmentType,
 )
+from .limits import OutputLimitExceededError, SandboxEnvironmentLimits
 from .local import LocalSandboxEnvironment  # noqa: F401
 from .registry import sandboxenv
 
 __all__ = [
+    "OutputLimitExceededError",
     "SandboxEnvironment",
+    "SandboxEnvironmentLimits",
```

Comment:
I don't think we need `SandboxEnvironmentLimits` exported, do we?

Reply:
We'll want it for the `K8sSandboxEnvironment` (and any other 3rd party ones) - have created a PR! #790